Hyperledger FireFly Labs Proposal

nguyer opened the pull request.

lehors reviewed (changes requested):
LGTM but you forgot to rename the file. You don't want to change the proposal-template.md file. Instead you want to use that file to create a new file called firefly.md
Thanks.

nguyer commented:
> LGTM but you forgot to rename the file. You don't want to change the proposal-template.md file. Instead you want to use that file to create a new file called firefly.md
> Thanks.

Thanks for pointing that out! I renamed it and moved it to the `./labs` directory now.

lehors reviewed (approved):
LGTM. Thanks.

tkuhrt commented on labs/firefly.md:
```diff
@@ -0,0 +1,30 @@
+- https://github.com/kaleido-io/firefly
+- https://github.com/kaleido-io/firefly-cli
+- https://github.com/kaleido-io/firefly-ui
+- https://github.com/kaleido-io/ethconnect
```
Will we be renaming this repository to something like firefly-ethconnect?

Also, will we have resolved licensing issues with this particular repo prior to transferring into labs?

nguyer replied:
Yes, we can rename this repo as a part of moving it over.

The licensing issues have been resolved in `ethconnect` now.

tkuhrt commented on labs/firefly.md:
```diff
@@ -0,0 +1,30 @@
+- https://github.com/gabriel-indik
+- https://github.com/shorsher
+- https://github.com/awrichar
+
```
In the template, there is a section for Sponsor. Can you add this section and note that you do not have a sponsor?

nguyer replied:
I do not see the string "Sponsor" in this file anywhere: https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/main/proposal-template.md

Am I missing something?

tkuhrt replied:
My bad. Looks like Ry removed it 4 days ago.

nguyer replied:
No worries! Thanks!

lehors replied:
> My bad. Looks like Ry removed it 4 days ago.

I think that's a mistake actually. We said the Sponsor was no longer required but we didn't eliminate the role.
Note: this has nothing to do with Firefly but I think the Sponsor section should be put back into the template. @ryjones ?

ryjones replied:
@lehors OK, sorry, I'm out over my skis. I'll put it back

tkuhrt reviewed (approved):
LGTM
